packages/slice-machine/lib/builders/CustomTypeBuilder/SliceZone/UpdateSliceZoneModalList.tsx

```diff
@@ -18,79 +16,27 @@ const UpdateSliceZoneModalList: React.FC<{
       <Grid
         gridTemplateMinPx="200px"
         elems={availableSlices}
-        defineElementKey={(slice: ComponentUI) => slice.model.name}
-        renderElem={(slice: ComponentUI) => {
-          const isPlaceholderSlice = placeholderSlices?.find(
-            (s) => s.model.id === slice.model.id
+        defineElementKey={(slice) => `${slice.from}-${slice.model.name}`}
+        renderElem={(slice) => {
+          const isPlaceholderSlice =
```

Comment:
⚠️ Since you introduce the naming `template` in this component, let's rename `placeholderSlices` to `sliceTemplates` so it will be simpler no?